readyplayerme/rpm-unity-sdk-core
# Feature/shader override update

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

-   This PR reworks the shader override functionality to support multiple property types, (Previously it was only textures). This enables the proper transfer of correct values like metallic and roughness






## How to Test

-  Add a shader override to an Avatar Config
- Enter in correct target property names (these can be found by selecting your shader and looking at the property names)
- assign the config in RPM Settings window
- Load an avatar from RPM Avatar loader window
- Check that values properly mapped



## Checklist
